src/commands/status/status.ts

```diff
@@ -37,16 +46,18 @@ export const status = async (options: OptionValues, command: BaseCommand) => {
     }
   }
 
-  const ghuser = command.netlify.globalConfig.get(`users.${current}.auth.github.user`)
+  const ghuser = command.netlify.globalConfig.get(`users.${current}.auth.github.user`) as string
```

Comment:
What guarantees that this is a string and not `undefined`?

Reply:
I believe there's some convoluted implicit guarantees (something like: if it failed to get a token/user, it wouldn't reach this code), but as above... I'll fix the type and deal with the fallout. 👍🏼